corehq/apps/userreports/tests/test_utils.py

```diff
@@ -43,7 +43,7 @@ def test_table_trickery(self):
 
     def test_long_table_name(self):
         name = get_table_name('this_is_a_long_domain', 'and_a_long_table_name')
-        name_expected = 'config_report_this_is_a_long_domain_and_a_long_table_n_6ac28759'
+        name_expected = '{}this_is_a_long_domain_and_a_long_table_n_6ac28759'.format(UCR_TABLE_PREFIX)
```

Comment:
This hash is going to change (the hash when the table name gets truncated includes the prefix)

Reply:
yes